routes/rooms.go

```diff
@@ -17,6 +19,22 @@ type GetSingleERResponse struct {
 	Departments []models.DepartmentBase
 }
 
+type paginate struct {
```

Comment:
Maybe move this to `routes/pagination.go` and add a comment with the meaning and usage of this struct